src/api/veilarbperson.ts

```diff
@@ -11,5 +11,5 @@ export interface MalformData {
 }
 
 export function fetchMalform(fnr: string): AxiosPromise<MalformData> {
-	return axiosInstance.get(`/veilarbperson/api/v2/person/malform?fnr=${fnr}`);
+	return axiosInstance.post(`/veilarbperson/api/v3/person/hent-malform`, { fnr });
```

Comment:
is it necessary to send body content as: `{fnr: fnr}`, or it works like this as well?

Reply:
json is "a little to smart"
som when the key and the valuevariabel is the same: `{ fnr: fnr }` equals `{ fnr }` , but if one of them was different you have to specify: `{ fnr: fodselsnummer }`... if that makes sence

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions